concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/GetAncestors.java

```diff
@@ -0,0 +1,46 @@
+package com.concordium.sdk.examples;
+
+import com.concordium.sdk.ClientV2;
+import com.concordium.sdk.Connection;
+import com.concordium.sdk.Credentials;
+import com.concordium.sdk.exceptions.BlockNotFoundException;
+import com.concordium.sdk.exceptions.ClientInitializationException;
+import com.concordium.sdk.requests.BlockHashInput;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.Callable;
+
+@Command(name = "GetAncestors", mixinStandardHelpOptions = true)
+public class GetAncestors implements Callable<Integer> {
```

Comment:
Please add one sentence describing the intention of the example.

Reply:
done